pulsar-broker-common/src/test/java/org/apache/pulsar/common/configuration/PulsarConfigurationLoaderTest.java

```diff
@@ -152,6 +159,25 @@ public void testPulsarConfiguraitonComplete() throws Exception {
         }
     }
 
+    @Test
+    public void testBackwardCompatibility() throws IOException {
+        File testConfigFile = new File("tmp." + System.currentTimeMillis() + ".properties");
+        if (testConfigFile.exists()) {
+            testConfigFile.delete();
+        }
+        try (PrintWriter printWriter = new PrintWriter(new OutputStreamWriter(new FileOutputStream(testConfigFile)))) {
+            printWriter.println("zooKeeperSessionTimeoutMillis=60");
```

Comment:
can you add another test where the configuration contains both zooKeeper* and metadataStore* properties ?

Reply:
@nicoloboschi I have changed the PR, could you please help review it again?